pytensor/graph/basic.py

```diff
@@ -907,6 +907,54 @@ def graph_inputs(
     yield from (r for r in ancestors(graphs, blockers) if r.owner is None)
 
 
+def explicit_graph_inputs(
+    graph: Variable[Any, Any] | Iterable[Variable[Any, Any]],
+) -> Generator[Variable, None, None]:
+    """
+    Get the inputs into PyTensor variables
+
+    Parameters
+    ----------
+    graph: TensorVariable
+        Output `Variable` instances from which to search backward through
+        owners.
+
+    Returns
+    -------
+        Tensor variables that are input nodes with no owner, in the order
```

Comment:
```suggestion
    -------
    iterable
        Tensor variables that are input nodes with no owner, in the order
```

giving a name to the returned objects is optional, but the type is not https://numpydoc.readthedocs.io/en/latest/format.html#returns, it would also not render correctly if skipping the type info line

Reply:
I understand. Thank you so much for the resource.